### DISCUSSION TITLE
Authentication required to access your repository

### DISCUSSION THREAD

**Discussion by PascalLugon** [cibseven/discussions#1] [Q&A] [answered]
Hello,
I was trying to build my SpringBoot app using your artifact. Apparently credentials are required to access your repo. How can I process?
BR
Pascal

**Comment by bzzzil** [comment, marked as answer]
Hello Pascal,

Please check that you use correct artifact version of CIB seven (`7.22.0-cibseven`). In docs sometimes there is misleading 1.0.0 artifact version.

And do not forget to use CIB seven repository (in `pom.xml` file, or in `settings.xml`):

```
  <repositories>
    <repository>
      <id>mvn-cibseven-public</id>
      <name>CIB seven Public Repository</name>
      <url>https://artifacts.cibseven.de/repository/public/</url>
    </repository>
  </repositories>
```

See the details here https://docs.cibseven.de/get-started/apache-maven/

Sadly, we still did not update all the documentation available. So there are still some places with misleading and erroneous information we are actively fixing right now. 

Regards, Vasily

**Comment by PascalLugon** [comment]
Vasily,
I've updated my pom.xml with the right version and repository.
But apparently your public repo is protected:
https://[repo.spring.io/plugins-release/org/camunda/bpm/springboot/camunda-bpm-spring-boot-starter-webapp/7.22.0-cibseven/camunda-bpm-spring-boot-starter-webapp-7.22.0-cibseven.pom](https://repo.spring.io/plugins-release/org/camunda/bpm/springboot/camunda-bpm-spring-boot-starter-webapp/7.22.0-cibseven/camunda-bpm-spring-boot-starter-webapp-7.22.0-cibseven.pom)
->
{
    "errors": [
        {
            "status": 401,
            "message": "Unauthorized"
        }
    ]
}
BR

**Comment by bzzzil** [comment]
Pascal, 

repo.spring.io is definitely not our public repo.

This artifact is expected to be taken from: https://artifacts.cibseven.de/repository/public/org/camunda/bpm/springboot/camunda-bpm-spring-boot-starter-webapp/7.22.0-cibseven/camunda-bpm-spring-boot-starter-webapp-7.22.0-cibseven.pom

And it should be available.

Looks like you have several extra repositories defined and not the right one was selected by maven.

You can try to change order of repositories or experiment with mirrors, like it is suggested here: https://stackoverflow.com/questions/34238196/how-maven-handle-multiple-repository-configurations

**Comment by PascalLugon** [comment]
Vasily,
You are right...
my local Maven settings were (in .m2) were causing the issue. I need to work the repositories order...
BR

**Comment by amardeep2006** [comment]
I did face similar issues . I saw a lots of 403 errors and had to add maven central as repo in my pom.xml.
I did not have any maven repo defined else where. 

```xml
  <repositories>
    <repository>
      <id>mvn-cibseven-public</id>
      <name>CIB seven Public Repository</name>
      <url>https://artifacts.cibseven.de/repository/public/</url>
    </repository>
    <repository>
      <id>central</id>
      <name>Maven Central Repository</name>
      <url>https://repo1.maven.org/maven2</url>
    </repository>
  </repositories>
```

**Reply by bzzzil** [reply]
Interesting. 403 errors on any cibseven artifact? Or maven is trying to get any classic artifact from cibseven repo and fail? 

I will check this on clean environment.
